packages/app-desktop/gui/ConfigScreen/controls/plugins/PluginsStates.tsx

```diff
@@ -326,3 +334,11 @@ export default function(props: Props) {
 		</Root>
 	);
 }
+
+const mapStateToProps = (state: AppState) => {
+	return {
+		pluginSearchQuery: state.route.props?.pluginId ?? '',
+	};
+};
+
+export default connect(mapStateToProps)(PluginsStates);
```

Comment:
I'd rather we don' connect deeply nested components, it makes it hard to manage. You could just pass `route.props` to all children or ConfigScreen